### PR TITLE
DOCS-14196 added .Net 5

### PR DESCRIPTION
**Pull request by kyuan-mongodb**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCS-14196

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/ccfcbef/drivers/docsworker-xlarge/DOCS-14196-added-Net5/csharp/

### CODE REVIEW

**Review by MongoCaleb — CHANGES_REQUESTED** [review]
I know this is an inherited problem, but I think we should take the time here to order the table so that the newest version is on the left.

**Review by MongoCaleb — APPROVED** [review]
LGTM!

File: source/includes/language-compatibility-table-csharp.rst [hunk 1]
```diff
@@ -5,17 +5,18 @@
...
+     - .NET 4.7
+     - .NET 4.6
+     - .NET 4.5
      - .NET Core 3.1
+     - .NET Core 3.0
+     - .NET Core 2.2
+     - .NET Core 2.1
+     - .NET Core 2.0
+     - .NET Core 1.1
+     - .NET Core 1.0
```
Comment (terakilobyte) [inline source/includes/language-compatibility-table-csharp.rst:19]: This ordering is incorrect. Strangely, the plain `.NET x` < 5 should follow after `.Net Core x`. See [here](https://www.tutorialsteacher.com/csharp/csharp-version-history). For reasons in the past we've omitted the word "Framework" from those early versions. I'm ok keeping this as nobody has complained, but we should keep them in order.

**Review by terakilobyte — CHANGES_REQUESTED** [review]
Please correct the order

**Pull request merged**